### DISCUSSION TITLE
메인페이지에 팝업 구성.

### DISCUSSION THREAD

**Discussion by chanhihi** [jiphyeonjeon-42/frontend#451] [Ideas]
![image](https://github.com/jiphyeonjeon-42/frontend/assets/85754295/7f16f7af-2132-4622-8721-d628af997c1b)

해당 페이지는 [국립 중앙 박물관](https://www.nl.go.kr/) 의 메인 페이지 입니다.

집현전 홈페이지 내에서 공지, 광고 등 기능을 할 수있는 팝업 기능을 제안합니다.

**Comment by scarf005** [comment]
클러스터에서 팝업 적용 여부에 대해 @chanhihi 님과 이야기를 나누어보았는데, 저는 좋다고 생각합니다.
1. 42 Wassup 홍보를 위해 필요하다고 하셨는데, 일주일 간 게시이므로 기간이 길지 않습니다.
2. 새 기능을 빠르게 적용하고 피드백을 받아 수정하는 것이 홈페이지 개발 속도를 올리는 데 도움을 줄 것 같습니다.

**Reply by chanhihi** [reply]
저도 빠르게 적용하고 싶은데.. 개인 사정으로 화요일까지 시간이 나질 않아서.. 혹시 @scarf005 께서 해당 기능을 맡아 주실 수 있으신가요 ?

**Reply by scarf005** [reply]
제가 일정이 바빠서 어려울 것 같습니다 죄송합니다...
